src/main/java/mil/dds/anet/resources/EventSeriesResource.java

```diff
@@ -0,0 +1,109 @@
+      throw new WebApplicationException("Event series not found", Status.NOT_FOUND);
+    }
+    return es;
+  }
+
+  @GraphQLQuery(name = "eventSeriesList")
+  @AllowUnverifiedUsers
+  public AnetBeanList<EventSeries> search(@GraphQLRootContext Map<String, Object> context,
+      @GraphQLArgument(name = "query") EventSeriesSearchQuery query) {
+    final Person user = DaoUtils.getUserFromContext(context);
+    query.setUser(user);
+    return dao.search(query);
+  }
+
+  @GraphQLMutation(name = "createEventSeries")
+  public EventSeries createEventSeries(@GraphQLRootContext Map<String, Object> context,
+      @GraphQLArgument(name = "eventSeries") EventSeries eventSeries) {
+    final Person user = DaoUtils.getUserFromContext(context);
+    validateEventSeries(user, eventSeries);
+
+    eventSeries.setDescription(Utils.isEmptyHtml(eventSeries.getDescription()) ? null
+        : Utils.sanitizeHtml(eventSeries.getDescription()));
+
+    final EventSeries created = dao.insert(eventSeries);
+
+    AnetAuditLogger.log("Event Series {} created by {}", created, user);
+    return created;
+  }
+
+  @GraphQLMutation(name = "updateEventSeries")
+  public Integer updateEventSeries(@GraphQLRootContext Map<String, Object> context,
+      @GraphQLArgument(name = "eventSeries") EventSeries eventSeries) {
+    final Person user = DaoUtils.getUserFromContext(context);
+    validateEventSeries(user, eventSeries);
+
+    eventSeries.setDescription(Utils.isEmptyHtml(eventSeries.getDescription()) ? null
+        : Utils.sanitizeHtml(eventSeries.getDescription()));
+
+    final int numRows = dao.update(eventSeries);
+    if (numRows == 0) {
+      throw new WebApplicationException("Couldn't process event series update", Status.NOT_FOUND);
+    }
+
+    AnetAuditLogger.log("EventSeries {} updated by {}", eventSeries, user);
+    // GraphQL mutations *have* to return something, so we return the number of updated rows
+    return numRows;
+  }
+
+  private void validateEventSeries(final Person user, final EventSeries eventSeries) {
+    if (eventSeries.getName() == null || eventSeries.getName().trim().isEmpty()) {
+      throw new WebApplicationException("Event Series name must not be empty", Status.BAD_REQUEST);
+    }
+    if (eventSeries.getDescription() == null || eventSeries.getDescription().trim().isEmpty()) {
+      throw new WebApplicationException("Event Series description must not be empty",
+          Status.BAD_REQUEST);
+    }
+    if (eventSeries.getHostOrgUuid() == null || eventSeries.getHostOrgUuid().trim().isEmpty()) {
+      throw new WebApplicationException("Event Series Host Organization must not be empty",
+          Status.BAD_REQUEST);
+    }
+    if (eventSeries.getAdminOrgUuid() == null || eventSeries.getAdminOrgUuid().trim().isEmpty()) {
+      throw new WebApplicationException("Event Series Admin Organization must not be empty",
+          Status.BAD_REQUEST);
+    }
+    assertPermission(user, eventSeries.getAdminOrgUuid());
```

Comment:
When updating you should check that the user has permission for the **original** adminOrg, as well as the one the user passed in the GraphQL mutation!